packages/block-library/src/query-title/index.php

```diff
@@ -38,13 +38,23 @@ function render_block_core_query_title( $attributes ) {
 		}
 	}
 	if ( $is_search ) {
-		$title = __( 'Search results' );
+		$title = isset( $attributes['searchResultsTerm'] ) && ! empty( $attributes['searchResultsTerm'] ) ? $attributes['searchResultsTerm'] : __( 'Search results' );
```

Comment:
Calling `isset()` is not needed since `empty()` already checks if the property is set. Calling `!empty( $attributes['searchResultsTerm'] )` is equivalent to calling `isset( $attributes['searchResultsTerm'] ) && $attributes['searchResultsTerm']`.
```suggestion
		$title = ! empty( $attributes['searchResultsTerm'] ) ? $attributes['searchResultsTerm'] : __( 'Search results' );
```

Reply:
Thanks @anton-vlasenko.  I have fixed it in the latest commit :bow